### ISSUE TITLE
Uncaught reference error with Monix 3.2.0 and Scala-Js 1.0.1

### ISSUE THREAD

**Issue by kbielefe** [monix/monix#1166]
I am getting an `Uncaught ReferenceError: setImmediate is not defined` on [this line](https://github.com/monix/monix/blob/3409f0a3063f1dc5085644282b3b61dd241fad28/monix-execution/js/src/main/scala/monix/execution/schedulers/StandardContext.scala#L38) when running under Monix 3.2.0 and Scala-Js 1.0.1. Appears to be a manifestation of [this runtime behavior change](https://www.scala-js.org/news/2020/02/25/announcing-scalajs-1.0.0/#accessing-a-member-that-is-not-declared-causes-a-referenceerror-to-be-thrown). Should be able to replace it with:

 ```scala
if (js.typeOf(js.Dynamic.global.setImmediate) != "undefined")
```

A search reveals no other instances of this same idiom, but it may be worth checking for some of the other [breaking changes](https://www.scala-js.org/news/2020/02/25/announcing-scalajs-1.0.0/#breaking-changes).

**Comment by alexandru** [comment]
Thanks for the very detailed report @kbielefe, we are using `setImmediate` where available because `setTimeout` ends up being throttled, unfortunately the Node.js we are using in tests does have `setImmediate` available, therefore we did not catch this breaking change.

We'll fix it today and release and emergency release.

**Comment by kbielefe** [comment]
Thanks for the quick response. I should have mentioned this was while running in a browser/V8. For the record: 

Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36 OPR/67.0.3575.137

**Comment by Avasil** [comment]
@kbielefe @adamw Could you try `3.3.0-a855f81-SNAPSHOT`

**Comment by kbielefe** [comment]
Be happy to. How can I get the SNAPSHOT artifact?

**Comment by adamw** [comment]
@Avasil works for sttp-client :)

**Comment by adamw** [comment]
@kbielefe you need to add `resolvers += Resolver.sonatypeRepo("snapshots")`

**Comment by kbielefe** [comment]
@Avasil works great with my code.

**Comment by Avasil** [comment]
Awesome, thank you for feedback and reports!

I have just released official `3.2.1` with this fix
